README.md

```diff
@@ -155,6 +156,32 @@ A similar process can be used to build a cluster with TLS in one execution as to
   3. `install-certs.yml` - Installs the certificate and also applies the `redpanda_broker` role to the cluster nodes. Note: This will install and start Redpanda (and restart any brokers that do not have `skip_node=true` set)
 5. If `install-certs.yml` was not run in step iii above, you will need to run `provision-node.yml` which will install the `redpanda_broker` role. **Note: If TLS is enabled on the cluster, make sure that `-e tls=true` is set, otherwise this playbook will disable TLS across any nodes that don't have `skip_nodes=true` set.**
 
+## Upgrading a cluster
```

Comment:
how does this handle a downgrade? I'm assuming we depend upon redpanda itself to do the right thing if the version goes lower. But, is there some sort of gating we should do within the ansible to prevent going too far back?

Reply:
I think the tldr is that we don't really handle a downgrade ATM. It may work (surprised if it did) but it should be considered unsupported

Reply:
So, looks like we might be able to do this with the package_facts module, a version test to compare, and assert to fail the play.
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_facts_module.html https://docs.ansible.com/ansible/latest/collections/ansible/builtin/version_test.html
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/assert_module.html